vllm/worker/neuron_model_runner.py

```diff
@@ -75,6 +76,15 @@ def __init__(
 
         # Lazy initialization.
         self.model: nn.Module  # initialize after load_model.
+        self.model_config.generation_config = GenerationConfig(
```

Comment:
We call this `sampling_params` or `SamplingParams` in vLLM. It would be better if we can make it consistent.

Reply:
ack, update it.

Reply:
just to be clear the sampling params here is for neuron on-device specifically, so rename it as `neuron_sampling_params`.